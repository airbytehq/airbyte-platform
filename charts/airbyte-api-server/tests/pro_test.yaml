values:
  - ../../test_resources/pro/values.yaml
set:
  airbyteYml: 'path'
templates:
  - deployment.yaml
  - service.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}
