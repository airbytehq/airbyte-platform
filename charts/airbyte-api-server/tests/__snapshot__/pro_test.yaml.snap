manifest should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: airbyte-api-server
        app.kubernetes.io/version: 0.50.19
        helm.sh/chart: airbyte-api-server-0.47.26
      name: RELEASE-NAME-airbyte-api-server
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: airbyte-api-server
      strategy:
        type: Recreate
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: airbyte-api-server
        spec:
          containers:
            - env:
                - name: INTERNAL_API_HOST
                  valueFrom:
                    configMapKeyRef:
                      key: INTERNAL_API_HOST
                      name: RELEASE-NAME-airbyte-env
                - name: AIRBYTE_API_HOST
                  valueFrom:
                    configMapKeyRef:
                      key: AIRBYTE_API_HOST
                      name: RELEASE-NAME-airbyte-env
                - name: AIRBYTE_VERSION
                  valueFrom:
                    configMapKeyRef:
                      key: AIRBYTE_VERSION
                      name: RELEASE-NAME-airbyte-env
              image: airbyte/airbyte-api-server:0.50.19
              imagePullPolicy: IfNotPresent
              livenessProbe:
                failureThreshold: 3
                httpGet:
                  path: /health
                  port: http
                initialDelaySeconds: 60
                periodSeconds: 10
                successThreshold: 1
                timeoutSeconds: 1
              name: airbyte-api-server
              ports:
                - containerPort: 8006
                  name: http
                  protocol: TCP
              readinessProbe:
                failureThreshold: 3
                httpGet:
                  path: /health
                  port: http
                initialDelaySeconds: 30
                periodSeconds: 10
                successThreshold: 1
                timeoutSeconds: 1
              resources:
                limits: {}
                requests: {}
              volumeMounts: null
          serviceAccountName: airbyte-admin
          volumes: null
  2: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: airbyte-api-server
        app.kubernetes.io/version: 0.50.19
        helm.sh/chart: airbyte-api-server-0.47.26
      name: RELEASE-NAME-airbyte-api-server-svc
    spec:
      ports:
        - name: http
          port: 8006
          protocol: TCP
          targetPort: http
      selector:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: airbyte-api-server
      type: ClusterIP
